### ISSUE TITLE
Accessibility: EbaySectionNotice section not being announced by screen readers

### ISSUE THREAD

**Issue by HenriqueLimas** [eBay/ebayui-core-react#74]
EbaySectionNotice section region (role="region") is linking to a description empty element( id="section-notice-attention" element only contains a <svg>);

<img width="810" alt="Screen Shot 2022-08-29 at 12 50 34 PM" src="https://user-images.githubusercontent.com/2222191/187252837-d1aa496f-6fa8-4bba-9086-c6b85f7394c4.png">

The expected behavior is: "The region has an accessible name that it announced when the region is focused." - while currently the region is not announced by screen readers.

**Comment by HenriqueLimas** [comment]
Passing the aria-label on the EbaySectionNotice, will add an aria-label to the svg and screen reader will read it properly.
